iOS/Flows/Conversation/Conversation List/ConversationListViewController+Extensions.swift

```diff
@@ -66,8 +66,7 @@ extension ConversationListViewController {
             }.store(in: &self.cancellables)
 
         self.messageInputAccessoryView.textView.$inputText.mainSink { [unowned self] text in
-            guard let cid = self.getCurrentMessageSequence()?.streamCID else { return }
-            let conversationController = ChatClient.shared.channelController(for: cid)
```

Comment:
I accidentally introduced this bug recently. When sending typing events they have to be done on the same controller each time otherwise we'll start hitting our api request limit.